README.md

```diff
@@ -67,13 +74,10 @@ End a word with a . if you want to select an emoji if there are multiple
 matches, otherwise the first match will be picked. Type 'q' to exit.
 > q
 Bye
-
-
-# running the tests
-(venv) $ pip install -r requirements-dev.txt
-(venv) $ pytest
 ```
 
+### Easy of use: make a shell alias
```

Comment:
"Ease of use"

Reply:
Thanks for the spelling fixes, committing those now